### DISCUSSION TITLE
Version 0.5.0 on grafana.com

### DISCUSSION THREAD

**Discussion by verdel** [flant/grafana-statusmap#262] [General]
Can you please update plugin version on grafana.com to 0.5.0?

**Comment by shurup** [comment]
Sure! We sent our request to the official directory on April 7th (the same day we did this release). What we can do now is wait for the formal approval from Grafana maintainers. It should happen soon (hopefully).

**Reply by malmor** [reply]
@verdel version 0.5.0 is still missing on grafana.com, but the newer version 0.5.1 is available now:
![image](https://user-images.githubusercontent.com/62105800/168287436-8e2a61b6-1325-476e-b783-8b5ff01db475.png)
https://github.com/flant/grafana-statusmap/releases/tag/v0.5.1 🎉
